bnn_hmc/utils/data_utils.py

```diff
@@ -0,0 +1,298 @@
+# coding=utf-8
+# Copyright 2021 The Google Research Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Data loaders."""
+
+from typing import Generator, Tuple
+
+import jax
+import numpy as onp
+from jax import numpy as jnp
+import tensorflow as tf
+import tensorflow_datasets as tfds
+from tensorflow.keras.datasets import imdb
```

Comment:
Could you replace these tensorflow imports with the more internal-compatible:
```
imdb = tf.keras.datasets.imdb
sequence = tf.keras.preprocessing.sequence
```
Thanks!